viral/offline_sequence_detection.py

```
@@ -0,0 +1,1306 @@
+
+    sns.despine()
+    plt.tight_layout()
+    plt.savefig(
+        SERVER_PATH
+        / "viral_plots"
+        / "sequence_en_bloc"
+        / f"{mode}_weighted_r_trajectories.png"
+    )
+
+
+def plot_correlation_across_stages(mode: Literal["linear", "circular"]) -> None:
```

Ruff (0.14.10)

Redefinition of unused `plot_correlation_across_stages` from line 965

(F811)